app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_count)
+  end
```

Comment:
**OPTIONAL**

- [ ] Since you are using `after_save: update_comments_counter` it is usually good practice to make the method itself private.